### PR TITLE
Chore/fix title text

### PR DESCRIPTION
**Pull request by alexanderleegs**
Changes the title to `IsomerCMS` instead of `Isomer CMS`.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Isomer CMS</title>
```
Comment (harishv7) [inline public/index.html:32]: why do we not have the space? Isomer and CMS are 2 different words right?

Comment (alexanderleegs) [reply public/index.html:32]: Yup, but to be consistent with our comms - see [this thread](https://opengovproducts.slack.com/archives/C04S8QA6ES3/p1693875975226159)

**Comment by dcshzj** [comment]
@alexanderleegs nit: the `AllSitesHeader.tsx` and `Header.jsx` files contains the `alt` attribute that should also have the space removed, so we can be consistent. Same as the `VerifyUserDetailsModal.jsx` file.

**Comment by alexanderleegs** [comment]
> @alexanderleegs nit: the `AllSitesHeader.tsx` and `Header.jsx` files contains the `alt` attribute that should also have the space removed, so we can be consistent. Same as the `VerifyUserDetailsModal.jsx` file.


Good catch, fixed in https://github.com/isomerpages/isomercms-frontend/pull/1472/commits/0e14bca2ee63d5dd9289c963ccd9830ac9aa3566

**Review by dcshzj — APPROVED** [review]

**Pull request merged**